### DISCUSSION TITLE
'chia keys add' example?

### DISCUSSION THREAD

**Discussion by dorkmo** [Chia-Network/chia-blockchain#1836] [Q&A] [answered]
could someone give an example of how to enter keys into the command line?
"Ener the mnemoic you want to use:"
one word then 'enter' at a time? seperated by commas? or just spaces? dont want to mess up lol thanks!

**Comment by hoffmang9** [comment, marked as answer]
It is just spaces and in the correct order IIRC.

**Reply by dorkmo** [reply]
thanks! worked great. left to right! :P
